### DISCUSSION TITLE
Nombre maximum de participants ?

### DISCUSSION THREAD

**Discussion by ChristianSimonSU** [seb-cogez/QCMCam#2] [Q&A]
Bonjour !

Je découvre QCMCam et je me pose plein de questions !
Tout d'abord quel est le nombre maximal de participants ? Est-ce limité par la définition des QR codes distribués ?

**Comment by seb-cogez** [comment]
Le nombre maximal actuel est ... euh je crois que j'ai adapté pour 90 personnes suite à une demande d'un universitaire, mais les limitations de la captation d'image vont vite empêcher un tel usage. Vous pouvez récupérer 60 marqueurs de vote sur le site, ce qui est déjà plus qu'assez je pense.
Bien cordialement

**Reply by ChristianSimonSU** [reply]
Effectivement, 60 c'est déjà un grand groupe !

Merci.
